code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2429,3 +2429,45 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	if(islizard(M))
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "triumph", /datum/mood_event/memories_of_home, name)
 	..()
+//TODO: Give some special things to this drink
+//I invite any coder to add some special reactions
+//or some shit like that, i dunno.
+/datum/reagent/consumable/ethanol/total_overreaction
+	name = "Total Overreaction"
+	description = "ARE YOU CALLING ME A LIAR SIR?!?!?"
+	boozepwr = 65
+	color = "#D34848"
+	taste_description = "your head growing red and cloudy with rage"
+	glass_icon_state = "total_overreaction"
+	glass_name = "Total Overreaction"
+	glass_desc = "ARE YOU CALLING ME A LIAR SIR?!?!?"
+/datum/reagent/consumable/ethanol/obscure_reference
+	name = "Obscure Reference"
+	description = "Originally made when some deserted people in a wasteland made a beer facility."
+	boozepwr = 25
+	color = "#000000"
+	taste_description = "ghetto but amazing beer"
+	quality = DRINK_NICE
+	glass_icon_state = "obsc_ref"
+	glass_name = "Obscure Reference"
+	glass_desc = "This drink has a label on the side: Strategic Nuclear Moose"
+/datum/reagent/consumable/ethanol/recurring_gag
+	name = "Recurring Gag"
+	description = "Who in their right mind would make this?"
+	boozepwr = 91
+	color = "#BFE600"
+	taste_description = "god's mistake"
+/datum/reagent/consumable/ethanol/recurring_gag/on_mob_life(mob/living/carbon/M)
+	if(rand(0, 3)==2)
+		M.vomit()
+/datum/reagent/consumable/ethanol/dub
+	name = "Dub"
+	description = "Originally created by a bored wizard who wanted to fuck with language spells."
+	boozepwr = 30
+	color = "#E5D824"
+	taste_description = "the wild, wild west"
+/datum/reagent/consumable/ethanol/dub/on_mob_add(mob/living/carbon/M)
+	M.grant_language(/datum/language/moffic)
+/datum/reagent/consumable/ethanol/dub/on_mob_delete(mob/living/carbon/M)
+	M.remove_language(/datum/language/moffic)
```

Comment:
It will forcefuly remove language from everyone. Including people that had it without the reagent.

Reply:
Oh..... Oh dear....